Corona-Warn-App/src/main/res/layout/include_onboarding.xml

```diff
@@ -180,19 +188,39 @@
                 app:layout_constraintStart_toStartOf="@+id/guideline_card_start"
                 app:layout_constraintTop_toBottomOf="@+id/onboarding_location_card" />
 
+            <include
+                layout="@layout/include_interoperability"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_normal"
+                android:layout_marginBottom="@dimen/spacing_normal"
+                android:visibility="@{FormatterHelper.formatVisibility(interopViewModel != null)}"
+                app:countryListTitle="@{@string/interoperability_onboarding_list_title}"
+                app:interopViewModel="@{interopViewModel}"
+                app:isOnboarding="@{true}"
+                app:layout_constraintBottom_toTopOf="@+id/onboarding_card"
+                app:layout_constraintEnd_toStartOf="@+id/guideline_card_end"
+                app:layout_constraintStart_toStartOf="@+id/guideline_card_start"
+                app:layout_constraintTop_toBottomOf="@+id/onboarding_location_card_16_years"
+                app:subTitle="@{@string/interoperability_onboarding_subtitle}"
+                app:subTitle2="@{@string/interoperability_onboarding_subtitle2}"
```

Comment:
also rename this if you change the text view id to be more consistent & of course only if it makes sense.